### PR TITLE
fix CI manifest

### PR DESCRIPTION
**Pull request by jpinsonneau**
Manifest image needs to be pushed to created the shortlived image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -316,7 +316,7 @@ else
 endif
 
 .PHONY: ci-manifest-build
-ci-manifest-build: manifest-build ## Build CI manifest
+ci-manifest-build: manifest ## Build CI manifest
```
Comment (jotak) [inline Makefile:319]: if "ci-manifest-build" runs "manifest" which itself does a push, then "ci-manifest-push" is now useless (or overlapping)?

Comment (jpinsonneau) [reply Makefile:319]: but it works like this in other components:
- `make images` build and push manifest
- `make ci-manifest` overrides the manifest shortlived

File: Makefile [hunk 2]
```diff
@@ -316,7 +316,7 @@ else
 endif
 
 .PHONY: ci-manifest-build
-ci-manifest-build: manifest-build ## Build CI manifest
+ci-manifest-build: ## Build CI manifest
```
Comment (msherif1234) [inline Makefile:319]: do we need to do the same for all other repos too ?

Comment (jpinsonneau) [reply Makefile:319]: :heavy_check_mark: yes !

File: .github/workflows/push_image_pr.yml [hunk 3]
```diff
@@ -41,8 +41,10 @@ jobs:
         run: IMAGE_ORG=${{ env.ORG }} IMAGE=${{ env.REGISTRY }}/${{ env.IMAGE }}:${{ env.short_sha }} make image-build
       - name: push images
         run: IMAGE_ORG=${{ env.ORG }} IMAGE=${{ env.REGISTRY }}/${{ env.IMAGE }}:${{ env.short_sha }} make image-push
-      - name: build and push manifest
-        run: IMAGE_ORG=${{ env.ORG }} IMAGE=${{ env.REGISTRY }}/${{ env.IMAGE }}:${{ env.short_sha }} make ci-manifest
+      - name: build temp manifest
```
Comment (jotak) [inline .github/workflows/push_image_pr.yml:44]: I'm trying these steps locally, it fails because it looks for image tags like `temp-amd64`

**Comment by jpinsonneau** [comment]
Replaced by https://github.com/netobserv/network-observability-operator/pull/342

**Pull request closed without merging**